### ISSUE TITLE
Can't pull container: dangerzone.rocks: invalid status code from registry 404 (Not Found)

### ISSUE THREAD

**Issue by franc00018** [freedomofpress/dangerzone#152]
I can't get dangerzone to run on Manjaro Linux. Details with `inxi`

```
System:
  Host: francois-xps8930 Kernel: 5.15.25-1-MANJARO x86_64 bits: 64
    Desktop: KDE Plasma 5.24.2 Distro: Manjaro Linux
```

I installed with `yay -Sy podman dangerzone` which completed successfully

Then I touched this file because it says it doesn't exist and crashed if I don't do it.

```bash
sudo touch /usr/share/dangerzone/image-id.txt
```

When I run dangerzone-cli, it crashes because it can't pull the container

```bash
dangerzone-cli ~/Téléchargements/50-0534-SO-eduquer-au-numerique.pdf
```

```
> /usr/bin/podman run --network none -v '/home/francois/Téléchargements/50-0534-SO-eduquer-au-numerique.pdf:/tmp/input_file' -v /home/francois/.config/dangerzone/tmp/tmp00yozwdb/pixels:/dangerzone dangerzone.rocks/dangerzone /usr/bin/python3 /usr/local/bin/dangerzone.py document-to-pixels
Invalid JSON returned from container: Trying to pull dangerzone.rocks/dangerzone:latest...

Invalid JSON returned from container: Error: initializing source docker://dangerzone.rocks/dangerzone:latest: pinging container registry dangerzone.rocks: invalid status code from registry 404 (Not Found)

documents-to-pixels failed
```

**Comment by Rojikku** [comment]
@franc00018 Hey, I maintain the copy of the package you are trying to install.
It is not working because I couldn't get podman to work, so I gave up testing, thinking the issue was only podman.

Technically I was right, but really the issue is because this repository currently wants you to build your own image. They do not host one.

If you download their repo, it is possible to build the image yourself and solve this problem.
You basically just have to run their script from here https://github.com/firstlookmedia/dangerzone/blob/v0.3.1/install/linux/build-image.sh and then move the final gz file and txt file into the share directory you mentioned above.

I would love to automate this for you, unfortunately, I cannot. I mentioned this issue above explaining the core of it.
I even attempted to make a version of the PKGBUILD that can only be built with the makepkg command to avoid the chroot issue, but it still gives permission errors. I do not know why because manually moving there and running the same commands works fine.

Until someone finds a solution for the above issue, I cannot fix this. Personally, I don't trust anyone else, including myself, to host the docker image for this container either.

**Comment by franc00018** [comment]
Thanks, it works. I just symlinked the setup from the Git repository. Nice software BTW !

**Comment by deeplow** [comment]
I'll close this issue as it's not an officially supported build. @Rojikku where should people follow up on this discussion? Arch forums?

**Comment by deeplow** [comment]
I am getting this issue on the newly released `v0.3.2` and but whereas this is a packaging issue, the one I'm getting is probably related to https://github.com/freedomofpress/dangerzone/issues/193 so I'll detail the issue there instead of here.

**Comment by Rojikku** [comment]
> I'll close this issue as it's not an officially supported build and was not built from source. @Rojikku is there a place where people can follow up on this discussion? Arch forums?

Eh sorry I didn't get dinged for this.
Technically you can comment on the AUR package. I'm not sure about the forums, I don't really use them.

Also, if dangerzone at any point wants control of the AUR package and I have some level of confirmation that's who I'm handing it to, I'm more than comfortable giving that to you guys so you can maintain and keep positive control over your distribution streams. Especially considering the complexity and value in integrity within this package.
